ReactSkia/components/RSkComponent.cpp

```diff
@@ -93,6 +94,15 @@ RnsShell::LayerInvalidateMask RSkComponent::updateProps(const ShadowView &newSha
       /*TODO : To be tested and confirm updateMask need for this Prop*/
       updateMask =static_cast<RnsShell::LayerInvalidateMask>(updateMask | RnsShell::LayerInvalidateAll);
    }
+   if(layer_->opacity && layer_->shadowOpacity && \
+      ( layer_->shadowColor != SK_ColorTRANSPARENT ) && \
+      (SkColorGetA(layer_->shadowColor) != SK_AlphaTRANSPARENT)) {
+        layer_->shadowFilter= SkImageFilters::DropShadowOnly( \
+                              layer_->shadowOffset.width(), layer_->shadowOffset.height(),\
+                              layer_->shadowRadius, layer_->shadowRadius,\
+                              layer_->shadowColor, nullptr);
+   }
```

Comment:
should there be a else condition for this ?
If shadowColor is set to transparent,layer->shadowFilter has to be reset ? Else shadowFilter will be using the old values itself right ?

Reply:
Yes, It a good catch. else needed